server/src/main/java/org/elasticsearch/index/IndexingPressure.java

```diff
@@ -265,11 +268,20 @@ public Releasable markReplicaOperationStarted(int operations, long bytes, boolea
 
     public boolean shouldSplitBulk(long size) {
         long currentUsage = (currentCombinedCoordinatingAndPrimaryBytes.get() + currentReplicaBytes.get());
-        return (currentUsage >= lowWatermark && size >= lowWatermarkSize) || (currentUsage >= highWatermark && size >= highWatermarkSize);
+        if (currentUsage >= highWatermark && size >= highWatermarkSize) {
+            highWaterMarkSplits.getAndIncrement();
+            logger.trace(() -> Strings.format("Split bulk due to high watermark: currentUsage [%d] and size [%d]", currentUsage, size));
```

Comment:
Perhaps this should either be labelled "current bytes". Or the `currentUsage` variable should be printed in a bytes representation.